hyde-themes.json

```diff
@@ -288,5 +288,15 @@
       "#161616",
       "#FF7EB6"
     ]
+  },
+  {
+    "THEME": "Ice Age",
```

Comment:
Test your local directory first before putting it in here 


` Hyde theme import "Theme Name" "/path/to/the/Theme"  `

You'll probably get some errors, as the Theme name you declared here is "Ice Age" 

but the file tree in your repo is "Ice-Age"
This will give errors

![image](https://github.com/user-attachments/assets/977a4b31-5751-499a-847c-06a2d6831fed)